### DISCUSSION TITLE
Page 91, regarding storage symbolic link

### DISCUSSION THREAD

**Discussion by AdamZWinter** [PracticalBooks/Practical-Laravel#3] [Ideas]
The wording in paragraph 3 suggests that you have to create the symbolic link, THEN run "php artisan storage:link" rather than that command creating for you.

**Comment by AdamZWinter** [comment]
Also, this is a problem:

https://stackoverflow.com/a/54713477/10664600

**Comment by danielgara** [comment]
Hi @AdamZWinter you are right. The discussion in that paragraph is not clear. I will fix it for a future version, thanks for reporting it!!
